### ISSUE TITLE
Zoom with mouse wheel + modifier not working anymore

### ISSUE THREAD

**Issue by Gabby16Bitbox** [fcpxhacks/fcpxhacks#2983]
All of the sudden i can't do use this function anymore "Zoom + Mouse Wheel", tried to reinstall change keys, but to no anvil.

**Comment by Gabby16Bitbox** [comment]
Reopening because it's still not working for me

**Comment by latenitefilms** [comment]
Ummm, it's definitely working fine here.

Have you already tried uninstalling and reinstalling CommandPost?

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/22286696/169960839-48b46dad-d9b4-424f-9d95-a143d19ae8a1.png">

**Comment by Gabby16Bitbox** [comment]
Yeah, i even tried to install older versions, but nothing. I have tried to uninstall some apps that i tought that could cause issues, but is not working. Tested on my other iMac and is working
Those are the apps i have installed:
![Schermata 2022-05-24 alle 10 19 01](https://user-images.githubusercontent.com/105720077/169984626-b22b81b8-46ec-4781-a32a-f23ad6939b16.png)
![Schermata 2022-05-24 alle 10 19 44](https://user-images.githubusercontent.com/105720077/169984696-4f4e260f-e5b6-4fe9-aeb0-caccbdd7fc58.png)
![Schermata 2022-05-24 alle 10 19 47](https://user-images.githubusercontent.com/105720077/169984718-b4c5d34b-38d9-49ef-8ccb-56fec589d09e.png)

I noticed that the problem appeared when i installed an app called Smooze Pro, for the wheel acceleration, but i uninstalled it, even deleted some hidden files in Library. I'm not sure it is this program tho. Probably an app causes issues, but i don't know which app.

My guesses are SteerMouse - Better Touch Tools (tried to uninstall it) - Smooze Pro
.
